src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+    UserList: 'lista de usuários',
+    uploading: 'Processando...',
+    userListHeader: 'Usuários ',
+    commonChatHeader: 'Sala de Bate-papo',
+    teacherVideoHeader: 'Vídeo do professor',
+    askQuestionHeader: 'Levantar Questão',
+    questionLable: 'Questão',
+    answerLable: 'Resposta',
+    commentLable: 'Comentário',
+    markAnswer: 'Marcar como Resposta',
+    markNotes: 'Marcações e Anotações',
+    askQuestionTimeExceed: 'Incapaz de editar/deletar pois você já excedeu o tempo limite',
+    upvoted: 'Incapaz de editar/deletar pois um voto a favor já foi adicionado',
+    enterText: 'Inserir o texto antes de postar',
+    more: '...Mais',
+    less: 'Menos',
+    askQuestion: 'Levantar questão',
+    MarksAndNotes: 'Marcações e Anotações',
+    markAnswerUnmark: 'Você precisa primeiro desmarcar a resposta já marcada',
+    /* For Chrome */
+    PermissionDeniedErrorExt: '<div class="errorMsg"> Acesso à câmera foi bloqueado. Para providenciar acesso à Webcam, favor seguir os procedimentos abaixo <br />'
+    + 'Vá para o ícone de câmera no topo direito da tela <br /> '
+    + 'Clique na opção "Sempre permitir..." e selecione a opção de câmera <br />  '
+    + 'Clique em finalizado e recarregue a tela  </div> <div class="screenImages">'
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-vídeo.png" > <figcaption>Chrome</figcaption> </figure></div>',
+
+    SecurityErrorExt: '<div class="errorMsg"> Acesso à câmera foi bloqueado. Para providenciar acesso à Webcam, favor seguir os procedimentos abaixo <br />'
+    + 'Vá para o ícone de câmera no topo direito da tela <br /> '
+    + 'Clique na opção "Sempre permitir..." e selecione a opção de câmera <br />  '
+    + 'Clique em finalizado e recarregue a tela  </div> <div class="screenImages">'
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-vídeo.png" > <figcaption>Chrome</figcaption> </figure></div>',
+
+    nopermissionExt: '<div class="errorMsg"> Acesso à câmera foi bloqueado. Para providenciar acesso à Webcam, favor seguir os procedimentos abaixo <br />'
+    + 'Vá para o ícone de câmera no topo direito da tela <br /> '
+    + 'Clique na opção "Sempre permitir..." e selecione a opção de câmera <br />  '
+    + 'Clique em finalizado e recarregue a tela  </div> <div class="screenImages">'
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-vídeo.png" > <figcaption>Chrome</figcaption> </figure></div>',
+
+    chromeExtMiss: "Congrea precisa do plugin 'Desktop Selector' para compartilhar a Tela.<br />"
+    + "Você pode baixá-lo <a href='https://chrome.google.com/webstore/detail/desktop-selector/ijhofagnokdeoghaohcekchijfeffbjl' target='_blank'>AQUI.</a>",
+
+
+    /* For Firefox */
+    PermissionDeniedErrorExtFF: '<div class="errorMsg"> Acesso à câmera foi bloqueado. Para providenciar acesso à Webcam, favor seguir os procedimentos abaixo <br />'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 166. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482325)